### DISCUSSION TITLE
Raspap AP "disapear" after first client successfully connect

### DISCUSSION THREAD

**Discussion by alexulb1** [RaspAP/raspap-webgui#1580] [Q&A]
I've setup raspap with wlan0 connected to internet router and wlan1 (external usb dongle Realtech RTL88x2bu [AC1200 Techkey] connect the wifi clients. After the first client was successfully connected to the AP and were able to access the internet as well, the AP disapear so other client can't see the AP and connect to it.
The first connected client is still connected to the AP and has connection to the internet as well which is strange because AP is not visible for not connected clients anymore.
Maybe somebody had the same issue in the past and solved it.
I'm using RPi5 with the latest raspberry os light 64 version and Raspap V. 3.1.3

**Comment by billz** [comment]
If the issue affects some clients, but not others, it's very likely related to the device(s) you're attempting to connect to the AP. On the other hand, if you suspect the problem is with your AP have you:

1. Surveyed your Wi-Fi neighborhood for overlapping SSIDs on the same or adjacent channels?
2. Tried connecting clients to the onboard adapter only? (ie, not the dongle)
3. Tried other channels?
4. Tried other wireless modes?
5. Enabled DHCP request logging?

You'll have a much higher probability of getting help (or perhaps answering the question yourself) by trying the above suggestions first.

**Reply by alexulb1** [reply]
Thanks for your support. I did some more investigation and it seems for me that the issue is independent of the client. I've tried it out with several clients (iphone, imac, Android phone, tablet and windows 11, but it's always the same. After reboot I see the AP and I'm able to connect and get access to the internet as well. As soon as I'm connected 1..2 min. later the AP disapear for not connected clients and also the connected client lost connection.
I've done all your recommended steps:
1. no overlapping SSIDs on that channel
2. switched wlan0 and wlan1 that clients connect to wlan0 instead of wlan1
3. I've tried other channels and switched wireless modes
4. I've enabled DHCP logging, but no errors apear !


The only two things which are different are the following:
1. logging shows some "disassociated due to inactivity" entries for wlan0 

wlan0: STA 3c:a6:f6:64:61:ae IEEE 802.11: disassociated
wlan0: STA 3c:a6:f6:64:61:ae IEEE 802.11: deauthenticated due to inactivity (timer DEAUTH/REMOVE)
wlan0: AP-STA-DISCONNECTED a2:96:9a:ca:0c:c7
wlan0: STA a2:96:9a:ca:0c:c7 IEEE 802.11: disassociated due to inactivity
wlan0: AP-STA-DISCONNECTED 3e:b2:68:d8:79:8a
wlan0: STA 3e:b2:68:d8:79:8a IEEE 802.11: disassociated due to inactivity
wlan0: STA a2:96:9a:ca:0c:c7 IEEE 802.11: deauthenticated due to inactivity (timer DEAUTH/REMOVE)
wlan0: STA 3e:b2:68:d8:79:8a IEEE 802.11: deauthenticated due to inactivity (timer DEAUTH/REMOVE)

2. even wlan0 is shown as up in raspap it is disapeared in the network setting
![grafik](https://github.com/RaspAP/raspap-webgui/assets/109351274/c0a95416-8bca-4a36-9782-d5c9f2397eda)
![grafik](https://github.com/RaspAP/raspap-webgui/assets/109351274/862fb14a-3c1e-47dc-88b6-3370ba92f59d)

**Reply by billz** [reply]
There are two different FAQs that address this specifically. Search the docs for 'disassociated' and perform a self-diagnosis. You'll also find a related linked issue in the FAQ, so you're not the first to ask this 😉
